Entities/Characters/Knight/KnightLogic.as

```diff
@@ -1613,25 +1568,8 @@ void onHitBlob(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@
 	}
 }
 
-
-void Callback_PickBomb(CBitStream@ params)
-{
-	CPlayer@ player = getLocalPlayer();
-	if (player is null) return;
-
-	CBlob@ blob = player.getBlob();
-	if (blob is null) return;
-
-	u8 bomb_id;
-	if (!params.saferead_u8(bomb_id)) return;
-
-	string matname = bombTypeNames[bomb_id];
-	blob.set_u8("bomb type", bomb_id);
-
-	blob.SendCommand(blob.getCommandID("pick " + matname));
-}
-
```

Comment:
Could you explain why this was changed? I do not see it mentioned in the PR

Reply:
It's because I made this PR before all those bunnie changes were submitted and merged, as is the case with many of my other PRs. It would take more time and dedication to sort these file conflicts out which I'm not really interested in doing for now.